src/cpp/server/insecure_server_credentials.cc

```diff
@@ -29,27 +29,36 @@ namespace grpc {
 namespace {
 class InsecureServerCredentialsImpl final : public ServerCredentials {
  public:
+  InsecureServerCredentialsImpl()
+      : creds_(grpc_insecure_server_credentials_create()) {}
+
+  ~InsecureServerCredentialsImpl() override {
+    grpc_server_credentials_release(creds_);
+  }
+
   int AddPortToServer(const std::string& addr, grpc_server* server) override {
     grpc_server_credentials* server_creds =
         grpc_insecure_server_credentials_create();
```

Comment:
This doesn't need to create a new creds object anymore; it can use `creds_` instead.

Reply:
Done